stix_shifter_modules/elastic_ecs/stix_translation/query_constructor.py

```diff
@@ -49,8 +49,10 @@ def _format_set(values) -> str:
 
     @staticmethod
     def _format_equality(value) -> str:
-
-        return '"{}"'.format(value)
+        if isinstance(value, str):
+            return '"{}"'.format(value.replace('\\', '\\\\'))
```

Comment:
@barvhaim kinda confused here. don't we have `_escape_value` function for this? Another question, does double backslash in the string makes an invalid query?

Reply:
@mdazam1942 The `if-cases` will go to the `_format_equality` case, the `_escape_value` does not get called. it makes incorrect query.